packages/amplify-category-function/package.json

```diff
@@ -35,7 +35,7 @@
     "folder-hash": "^4.0.2",
     "fs-extra": "^8.1.0",
     "globby": "^11.0.3",
-    "graphql-transformer-core": "^7.6.6",
+    "graphql-transformer-core": "8.0.0-beta.1",
```

Comment:
caret versions to be brought back in https://app.asana.com/0/1202389680978480/1203201977550369/f